bin/commands.dart

```diff
@@ -17,325 +13,26 @@ class Commands {
-          [
-            CommandOptionBuilder(
-                CommandOptionType.string, 'datum', 'YYYY-MM-dd',
-                required: true),
-          ],
-        )..registerHandler((event) async {
-            await event.acknowledge();
-            await _releaseCommand(event);
-          }),
-        SlashCommandBuilder(
-          'untappd',
-          'Let me know your untappd username so I can post automatic updates from your untappd account.',
-          [
-            CommandOptionBuilder(CommandOptionType.string, 'username',
-                'e.g. cornholio (kontot måste minst ha 1 incheckning)',
-                required: true),
-          ],
-        )..registerHandler((event) async {
-            await event.acknowledge();
-            await _untappdCommand(event);
-          }),
-        SlashCommandBuilder(
-          'setup',
-          'Setup the bot to post untappd updates to the current channel.',
-          [],
-          requiredPermissions: PermissionsConstants.administrator,
-        )..registerHandler((event) async {
-            await event.acknowledge();
-            await _setupUntappdServiceCommand(event);
-          }),
+        ...UntappdModule().commands,
       ];
 
   static Future<void> _helpCommand(ISlashCommandInteractionEvent ctx) async {
     var helpMessage = MessageBuilder()
       ..append(ctx.interaction.userAuthor!.mention)
       ..appendNewLine()
-      ..append('Did anyone say beer? This is what I can do for you:')
-      ..appendNewLine()
-      ..appendNewLine()
-      ..appendBold('/oel')
-      ..appendNewLine()
-      ..append('Lists all known beer releases.')
-      ..appendNewLine()
-      ..appendNewLine()
-      ..appendBold('/subscribe')
-      ..appendNewLine()
-      ..append(
-          'Subscribe to automatic beer release reminders. Reminders will be posted 3 times during the day before release.')
-      ..appendNewLine()
-      ..appendNewLine()
-      ..appendBold('/release YYYY-MM-dd')
-      ..appendNewLine()
-      ..append(
-          'Posts the beer release for given date in the format YYYY-MM-dd. e.g ')
-      ..appendItalics('/release 1970-01-30')
-      ..appendNewLine()
-      ..appendNewLine()
-      ..appendBold('/help')
+      ..append(_mainHelpMessage)
       ..appendNewLine()
-      ..append('Shows you this help message.')
       ..appendNewLine()
-      ..appendNewLine()
-      ..appendBold('/untappd untappd_username')
-      ..appendNewLine()
-      ..append(
-          'Let me know your untappd username so I can post automatic updates from your untappd account. e.g ')
-      ..appendItalics('/untappd cornholio')
-      ..appendNewLine()
-      ..appendNewLine()
-      ..appendBold('/setup')
-      ..appendNewLine()
-      ..append(
-          'Setup the bot to post untappd updates to the current channel. Only admins can issue this command. Also, this is needed before any untappd updates can occur.');
+      ..append(UntappdModule().helpMessage);
```

Comment:
**Suggestion:** Avoid creating multiple instances of `UntappdModule` in the code. Instead, create a single instance and reuse it. This will improve performance and memory usage.
```suggestion
        var untappdModule = UntappdModule();
        ...
        untappdModule.commands,
        ...
        untappdModule.helpMessage
```